.github/workflows/stale_issues.yml

```
@@ -0,0 +1,54 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    # Run at 2:30 AM every day
+    - cron: '30 2 * * *'
+  # Enable manual run
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # Set for dry-run only. TODO: remove debug-only
+          debug-only: true
+          # See discussion on generous time limits:
+          # https://github.com/shap/shap/discussions/3051
+          days-before-stale: 730  # 2 years
+          days-before-close: 90   # 3 months
+          stale-issue-message: "This issue has been inactive for two years, so it's
+            been automatically marked as 'stale'.
+            \n\n
+            We value your input! If you think this issue is still relevant, please
+            leave a comment to keep it open and remove the 'stale' label.
+            \n\n
+            If there's no activity in the next 30 days the issue will be closed."
```

```suggestion
            If there's no activity in the next 90 days the issue will be closed."
```

Or did I miss something? I thought line 24 configures this time span?!

---

Ah good spot, thanks